drivers/spi/Kconfig.xec_qmspi_v2

```diff
@@ -0,0 +1,12 @@
+# Microchip XEC QMSPI V2 supporting MEC172x
+
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config SPI_XEC_QMSPI_V2
+	bool "Microchip XEC QMSPI V2 driver"
+	default n
+	depends on SOC_SERIES_MEC172X
+	help
+	  Enable support for the Microchip XEC QMSPI driver
+	  for MEC172x.
```

Comment:
This could use an explanation as to what "V2" means? Why would someone want the non-V2 driver?

Reply:
updated in Kconfig.xec_qmspi_v2_ldma